src/evict/evict_page.c

```diff
@@ -549,6 +549,8 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
      */
     if (WT_PAGE_IS_INTERNAL(page)) {
         WT_WITH_PAGE_INDEX(session, ret = __evict_child_check(session, ref));
+        if (ret)
```

Comment:
`ret` is not boolean, we try to avoid integers being interpreted as boolean in the if conditions implicitly. This can be replaced with `ret != 0`

Reply:
Fixed.